zuul.d/project-templates.yaml

```diff
@@ -644,6 +644,20 @@
             required-projects:
               - name: github.com/ansible-collections/ansible.netcommon
 
+- project-template:
+    name: ansible-collections-security-ibm-qradar
+    check:
+      jobs:
```

Comment:
You need to add, this is the job that will prepare the `ibm.qradar` collection artifact that will be consume by ansible-security-integration-ibm-qradar:
`        - build-ansible-collection`